include/seqan3/alphabet/quality/composition.hpp

```diff
@@ -166,6 +167,16 @@ quality_composition(sequence_alphabet_type &&, quality_alphabet_type &&)
 
 } // namespace seqan3
 
+namespace seqan3::detail
+{
+
+//!\brief Since seqan3::quality_composition wraps a nucleotide alphabet it is also one.
+template <typename sequence_alphabet_type, typename quality_alphabet_type>
+struct is_nucleotide<quality_composition<sequence_alphabet_type, quality_alphabet_type>> : public std::true_type
+{};
```

Comment:
See the definition of  `quality_concept`: it requires the `to_phred` `assign_phred` ... stuff. Since this is provided, the type satisfies quality_concept (this is tested in another file).
`nucleotide_concept` is special, in that it needs to have an explicit metafunction set to true. That's because structurally it doesn't require more of the type, than e.g. aminoacid, i.e. no different member functions or types, but it represents something different (and it makes sense to restrict certain functions to one or the other).